hedera-platform/build.gradle.kts

```diff
@@ -30,7 +30,7 @@ val log4jVersion = "2.17.1"
 val mockitoVersion = "4.6.1"
 val nettyVersion = "4.1.87.Final"
 val protobufVersion = "3.21.7"
-val swirldsVersion = "0.42.0-adhoc.xfb77ea8d"
+val swirldsVersion = "0.42.0-adhoc.xf391383e"
```

Comment:
Changes in build.gradle.kts are probably not required for this fix, but may conflict with others' fixes.

Reply:
I'd have to update this anyway, it looks like